.changeset/witty-readers-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': minor
```

Comment:
This should be a `patch` release here, as `core-components` is still `0.x`

Reply:
Done!
Thanks